cli/smartcontract/smart_contract.go

```diff
@@ -130,27 +130,27 @@ func NewCommands() []cli.Command {
 				Flags: []cli.Flag{
 					cli.StringFlag{
 						Name:  "in, i",
-						Usage: "Input file for the smart contract to be compiled",
+						Usage: "Input file for the smart contract to be compiled (*.go file or directory). Omit all output-related flags to autocomplete",
```

Comment:
This feature doesn't affect input in any way, so I'm not sure what this message means to the user.